pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -367,15 +366,15 @@ var (
 	errInjectedFilterStatus = errors.New("injected filter status")
 )
 
-func newFrameworkWithQueueSortAndBind(r Registry, pl *config.Plugins, plc []config.PluginConfig, opts ...Option) (framework.Framework, error) {
+func newFrameworkWithQueueSortAndBind(r Registry, profile *config.KubeSchedulerProfile, opts ...Option) (framework.Framework, error) {
```

Comment:
Uhm... since we are always expecting a profile in this function, accept an object.